### DISCUSSION TITLE
Please add the scripts feature

### DISCUSSION THREAD

**Discussion by MonwF** [slackhq/nebula#1205] [Ideas]
like https://www.tinc-vpn.org/documentation/Scripts.html

**Comment by johnmaguire** [comment]
Can I ask what use case you have in mind?

**Reply by MonwF** [reply]
I desire to obtain [the feature](https://github.com/slackhq/nebula/issues/353) as a temporary solution. i can modify unsafe_routes when some node is down

**Reply by johnmaguire** [reply]
So your use case with the scripts feature is to detect when a specific host is inaccessible and then remove routes from the system? The more specific you are able to describe your use case, the more likely it is we can find and develop a solution to fit it. Thanks!

**Reply by MonwF** [reply]
thanks for your reply. imagine that i have two company networks, A with subnet 10.200.0.0/16, B 10.100.0.0/16 respectly. i would setup two hosts (marked as x and y) in network A as exit nodes for availbility. they both have been signed with subnet 10.200.0.0/16.  A host from network B is connected with the two nodes. At first i set unsafe_route to x. but when x is down, i want to change unsafe_route to y automaticly without change the system route
